consultation_analyser/consultations/models.py

```diff
@@ -1,7 +1,51 @@
-# Create your models here.
+import uuid
 from django.db import models
 
 
-class Question(models.Model):
+class UUIDPrimaryKeyBase(models.Model):
+    class Meta:
+        abstract = True
+
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+
+
+class SlugPrimaryKeyBase(models.Model):
+    class Meta:
+        abstract = True
+
+    slug = models.CharField(primary_key=True, blank=False, max_length=256)
+
+
+class Consultation(SlugPrimaryKeyBase):
+    name = models.CharField(max_length=256, blank=False)
```

Comment:
Per discussion: this should be unique (URLs will be sth like)

`consultations/{consultation-slug}/questions/{question-slug}`